MiniTwit.FlagTool.App/Program.cs

```diff
@@ -3,103 +3,93 @@
 using System.Data.SQLite;
 using System.Linq;
 
-namespace MiniTwit.FlagTool.App
+namespace MiniTwit.FlagTool

```

Comment:
Why has the namespace changed without the folder doing so? This does not follow our current standard.